src/resources/attachment.yaml

```diff
@@ -7,10 +7,44 @@ comment: |
   such as Dropbox or Google Drive.
 properties:
 
-  - name: id
-    <<: *PropType.Id
+  - name: created_at
+    <<: *PropType.DateTime
+    access: Read-only
+    comment: |
+      The time at which this attachment was uploaded.
+
+  - name: download_url
+    <<: *PropType.DownloadUrl
```

Comment:
so for example, I could understand having `PropType.Url` for any properties that are URLs. But maybe we'd end up overriding the example value for each one and so then the prop type template wouldn't be providing much value? You've been working in this more so you'd have a better understanding. If that is the case, I still think we want to change the name of the template from `PropType` to something else, since we really don't have this many distinct types of properties.